features/AssetCategory.feature

```diff
@@ -0,0 +1,189 @@
+      | categories | ["bigTruck"] |
+
+  Scenario: Create an asset with AssetCategory and absent mandatory metadatas
+    When I successfully execute the action "device-manager/metadata":"create" with args:
+      | engineId       | "engine-ayse" |
+      | body.name      | "surname"     |
+      | body.valueType | "integer"     |
+      | body.mandatory | true          |
+    When I successfully execute the action "device-manager/assetCategory":"linkMedatadata" with args:
+      | engineId    | "engine-ayse" |
+      | _id         | "truck"       |
+      | _metadataId | "surname"     |
+    When I execute the action "device-manager/asset":"create" with args:
+      | engineId        | "engine-ayse" |
+      | body.type       | "truck"         |
+      | body.model      | "M"             |
+      | body.reference  | "asset_02"      |
+      | body.categories | ["bigTruck"]    |
+      | body.metadata   | {}              |
+    Then I should receive an error matching:
+      | status | 400 |
+
+
+  Scenario: Create an asset with AssetCategory and present mandatory metadatas
+    When I successfully execute the action "device-manager/asset":"create" with args:
+      | engineId              | "engine-ayse" |
+      | body.type             | "truck"       |
+      | body.model            | "M"           |
+      | body.reference        | "asset_02"    |
+      | body.categories       | ["bigTruck"]  |
+      | body.metadata.surname | "test"        |
+
+
+  Scenario: Update a parent, and verify edition propagation
+    When I successfully execute the action "device-manager/assetCategory":"update" with args:
+      | engineId  | "engine-ayse" |
+      | _id       | "truck"       |
+      | body.name | "basicTruck"  |
+    Then The document "engine-ayse":"asset-category":"bigTruck" content match:
+      | name        | "bigTruck" |
+      | parent.name | "basicTruck"    |
+
+  Scenario: Unlink a metadata on a parent category, verify parent update and verify edition propagation on children category
+    When I successfully execute the action "device-manager/assetCategory":"unlinkMedatadata" with args:
+      | engineId       | "engine-ayse" |
+      | _id | "truck" |
+      | _metadataId | "surname"  |
+    Then The document "engine-ayse":"asset-category":"truck" content match:
+      | assetMetadatas | [] |
+    Then The document "engine-ayse":"asset-category":"bigTruck" content match:
+      | parent.assetMetadatas | [] |
+
+  Scenario: link a metadata on a parent category, and verify edition propagation on children category
+    When I successfully execute the action "device-manager/metadata":"create" with args:
+      | engineId       | "engine-ayse" |
+      | body.name      | "height"      |
+      | body.valueType | "integer"     |
+      | body.mandatory | false         |
+    When I successfully execute the action "device-manager/assetCategory":"linkMedatadata" with args:
```

Comment:
There is a typo here :grin:  linkMe**da**tadata

Reply:
^^".
Done :-)